camera/android/src/main/java/com/capacitorjs/plugins/camera/CameraPlugin.java

```diff
@@ -0,0 +1,602 @@
+package com.capacitorjs.plugins.camera;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.content.pm.PackageManager;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.MediaStore;
+import android.util.Base64;
+import androidx.core.app.ActivityCompat;
+import androidx.core.content.FileProvider;
+import com.getcapacitor.FileUtils;
+import com.getcapacitor.JSObject;
+import com.getcapacitor.Logger;
+import com.getcapacitor.NativePlugin;
+import com.getcapacitor.Plugin;
+import com.getcapacitor.PluginCall;
+import com.getcapacitor.PluginMethod;
+import com.getcapacitor.PluginRequestCodes;
+import com.getcapacitor.annotation.CapacitorPlugin;
+import com.getcapacitor.annotation.Permission;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The Camera plugin makes it easy to take a photo or have the user select a photo
+ * from their albums.
+ *
+ * On Android, this plugin sends an intent that opens the stock Camera app.
+ *
+ * Adapted from https://developer.android.com/training/camera/photobasics.html
+ */
+@CapacitorPlugin(
+    name = "Camera",
+    permissions = {
+        @Permission(permission = Manifest.permission.CAMERA, alias = "camera"),
+        @Permission(permission = Manifest.permission.WRITE_EXTERNAL_STORAGE, alias = "writePhotos"),
+        @Permission(permission = Manifest.permission.READ_EXTERNAL_STORAGE, alias = "readPhotos")
```

Comment:
Internal "permission" element is renamed to "strings" now as of todays update: ionic-team/capacitor#3822

```suggestion
        @Permission(strings = { Manifest.permission.CAMERA }, alias = "camera"),
        @Permission(strings = { Manifest.permission.WRITE_EXTERNAL_STORAGE }, alias = "writePhotos"),
        @Permission(strings = { Manifest.permission.READ_EXTERNAL_STORAGE }, alias = "readPhotos")
```